Python/pathconfig.c

```diff
@@ -566,9 +566,10 @@ Py_GetProgramName(void)
     return _Py_path_config.program_name;
 }
 
-/* Compute argv[0] which will be prepended to sys.argv */
-PyObject*
-_PyPathConfig_ComputeArgv0(const _PyWstrList *argv)
+/* Compute module search path which will be prepended to sys.argv:
```

Comment:
I know this function is private, but would it make sense to explain what the API is? Something like "It will return 0 if it fails to resolve the full path (and the result will be NULL) and will return 1 if the path is correctly resolved (but the result may be still NULL if the Unicode fail to be created).

Reply:
Ok, done.